ibm/resource_ibm_is_vpn_gateway.go

```diff
@@ -701,14 +769,14 @@ func isVpnGatewayDeleteRefreshFunc(vpnGateway *vpcv1.VpcV1, id string) resource.
 		getVpnGatewayOptions := &vpcv1.GetVPNGatewayOptions{
 			ID: &id,
 		}
-		vpngw, response, err := vpnGateway.GetVPNGateway(getVpnGatewayOptions)
+		_, response, err := vpnGateway.GetVPNGateway(getVpnGatewayOptions)
```

Comment:
These changes were done recently to fix testcase issues by vaishnavi...Please roll back them